examples/question-answering/run_qa_beam_search_no_trainer.py

```diff
@@ -0,0 +1,730 @@
+#!/usr/bin/env python
+# coding=utf-8
+# Copyright Shyam Sunder Kumar and The HuggingFace Inc. team. All rights reserved.
```

Comment:
```suggestion
# Copyright 2021 The HuggingFace Inc. team. All rights reserved.
```
The year is important.

Regarding the name, we usually only put organizations as the copyright holder, I think it comes to the template. GitHub history will remember forever that you are the awesome contributor who added this :-)